site/proxy.md

```diff
@@ -119,17 +119,12 @@ Containers started via the proxy are automatically registered in
 [weaveDNS](weavedns.html) if they have a hostname in the weaveDNS
 domain (usually `.weave.local`). In order for containers to be able to
 look up such names, their DNS resolver needs to be configured to point
-at weaveDNS. This can be done [manually](weavedns.html#without-run),
-or automatically by the proxy by launching it with the `--with-dns`
-option, i.e.
-
-    host1$ weave launch-proxy --with-dns
-
-Now any containers started via the proxy, and with a `WEAVE_CIDR=...`
-environment variable (or even without it, if IPAM is
-[configured](#ipam)), will use weaveDNS for name resolution. If not
-provided, the hostname will default to the container name and will be
-registered in weaveDNS.
+at weaveDNS. This will be done automatically by the proxy while
+weaveDNS is running. To override this behaviour launch the proxy with
+either `--with-dns` or `--without-dns`, which will force the proxy to
+always/never set the resolver to weaveDNS.  If not provided, the
+hostname will default to the container name and will be registered in
```

Comment:
that's not quite right; the hostname is set to container name in the weaveDNS domain.